### ISSUE TITLE
"npm run clean" does not work on Windows

### ISSUE THREAD

**Issue by NileshPatel17** [bdougie/open-sauced#358]
### Problem:
Below Script does not work on Windows
```
"clean": "rm -rf src/tests/__snapshots__/",
```
### Solution:
1. install ```rimraf``` as dev dependency
2. change script to
```
"clean": "rimraf src/tests/__snapshots__/",
```

Please let me know if i submit a PR.

**Comment by bdougie** [comment]
@NileshPatel17 great catch, I have not actually used windows during the development of this project. Please proceed with the PR so I can test on some environment.

Can you also sign up for the [Actions beta](https://github.com/features/actions/signup/), I believe your PR won't run the CI without it.

**Comment by NileshPatel17** [comment]
Will submit PR. do you want me to include ```package-lock.json``` as a part of PR as ```rimraf``` would be added as dependency?
I think, i already signed up for Action Beta. How can i know if i have sign up or not?

**Comment by bdougie** [comment]
>Will submit PR. do you want me to include package-lock.json as a part of PR as rimraf would be added as dependency?

yes, please include the package-lock.json the way the see works is it checks it for caching purposes.

> I think, i already signed up for Action Beta. How can i know if i have sign up or not?

When you navigate to the link it should not have a button that shows sign up and instead say you already have access.

**Comment by NileshPatel17** [comment]
submitted PR #370.

**Comment by NileshPatel17** [comment]
> > Will submit PR. do you want me to include package-lock.json as a part of PR as rimraf would be added as dependency?
> 
> yes, please include the package-lock.json the way the see works is it checks it for caching purposes.
> 
> > I think, i already signed up for Action Beta. How can i know if i have sign up or not?
> 
> When you navigate to the link it should not have a button that shows sign up and instead say you already have access.

alright, verified. I already have access.

**Comment by NileshPatel17** [comment]
> > > Will submit PR. do you want me to include package-lock.json as a part of PR as rimraf would be added as dependency?
> > 
> > 
> > yes, please include the package-lock.json the way the see works is it checks it for caching purposes.
> > > I think, i already signed up for Action Beta. How can i know if i have sign up or not?
> > 
> > 
> > When you navigate to the link it should not have a button that shows sign up and instead say you already have access.
> 
> alright, verified. I already have access.



> Will submit PR. do you want me to include package-lock.json as a part of PR as rimraf would be added as dependency?

found rimraf already part of  ```package.json```

**Comment by bdougie** [comment]
Addressed in #370
